documentation-site/pages/components/code.js

```diff
@@ -0,0 +1,65 @@
+import {LiveProvider, LiveEditor, LiveError, LivePreview} from 'react-live';
+// import { useHover } from "./hooks";
+// import {
+//   styled,
+//   withStyle,
+//   withTransform,
+//   createStyled,
+//   useStyletron
+// } from "styletron-react";
+// import { driver, getInitialStyle } from "styletron-standard";
+// import { DESKTOP_BREAKPOINT } from "../const";
+
+// import {Avatar} from '../../../src/avatar/avatar';
```

Comment:
3) This import caused an `Unexpected token **i**mport` error. The path above really goes to `src/avatar/avatar.js`. To solve this, I tried using `FILTERED` for the `scope`, in `transformScope`. It just threw a different `Unexpected token` error.

Reply:
yea, you can't pass imports / exports to `react-live`, it has some [limitations](https://github.com/FormidableLabs/react-live#what-code-can-i-use)